.erda/migrations/cmdb/20211102-cluster-scheduler-cover-tag.py

```diff
@@ -0,0 +1,58 @@
+"""
+Generated by Erda Migrator.
+Please implement the function entry, and add it to the list entries.
+"""
+import json
+
+import django.db.models
+from django.db.models import Q
+
+
+class CoClusters(django.db.models.Model):
+    """
+    generated by erda-cli
+    """
+
+    id = django.db.models.BigIntegerField()
+    org_id = django.db.models.BigIntegerField()
+    name = django.db.models.CharField()
+    display_name = django.db.models.CharField()
+    type = django.db.models.CharField()
+    cloud_vendor = django.db.models.CharField()
+    logo = django.db.models.TextField()
+    description = django.db.models.TextField()
+    wildcard_domain = django.db.models.CharField()
+    config = django.db.models.TextField()
+    urls = django.db.models.TextField()
+    settings = django.db.models.TextField()
+    scheduler = django.db.models.TextField()
+    opsconfig = django.db.models.TextField()
+    resource = django.db.models.TextField()
+    sys = django.db.models.TextField()
+    created_at = django.db.models.DateTimeField(auto_now=True)
+    updated_at = django.db.models.DateTimeField(auto_now=True, auto_now_add=True)
+    manage_config = django.db.models.TextField()
+
+    class Meta:
+        db_table = "co_clusters"
+
+
+def entry():
+    """
+    please implement this and add it to the list entries
+    """
+
+    condition_clusters = CoClusters.objects.filter(Q(type='k8s') | Q(type='edas'))
+
+    for cluster in condition_clusters:
+        fmt_scheduler = json.loads(cluster.scheduler)
```

Comment:
if the `cluster.scheduler` is empty, it will cause panic

Reply:
That does not currently exist.